pkg/util/clusterdata/ingress_profiles_task_test.go

```diff
@@ -0,0 +1,332 @@
+package clusterdata
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"reflect"
+	"testing"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+	operatorclient "github.com/openshift/client-go/operator/clientset/versioned"
+	fakeopclient "github.com/openshift/client-go/operator/clientset/versioned/fake"
+	"github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/fake"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/test/util/cmp"
+)
+
+func TestIngressProfilesEnricherTask(t *testing.T) {
+	log := logrus.NewEntry(logrus.StandardLogger())
+
+	oioNamespace := "openshift-ingress-operator"
+	oiNamespace := "openshift-ingress"
+	owningIngressLabel := "ingresscontroller.operator.openshift.io/owning-ingresscontroller"
+	for _, tt := range []struct {
+		name        string
+		operatorcli operatorclient.Interface
+		kubecli     kubernetes.Interface
+		wantOc      *api.OpenShiftCluster
+		wantErr     string
+	}{
+		{
+			name: "default simplest case of ingress profile found",
+			operatorcli: fakeopclient.NewSimpleClientset(
+				&operatorv1.IngressController{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      "simple",
```

Comment:
Is the befault one named `default`?  If so lets align names to real configuration